iothub/service/src/Microsoft.Azure.Devices.csproj

```diff
@@ -22,7 +22,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.16.0-preview-003</Version>
+    <Version>1.16.0-preview-004</Version>
```

Comment:
Why do we need to update the Service SDK Version?

Reply:
There were changes to the service done in master that were merged to modules-preview